docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -60,7 +60,7 @@ Here are 2 options:
 ```bash
 git clone https://github.com/NVIDIA/spark-rapids.git
 cd spark-rapids
-mvn -pl .,tools clean verify -DskipTests
+mvn -Pdefault -pl .,tools clean verify -DskipTests
```

Comment:
I guess this is fine, but we haven't updated any docs because build stuff is very much in flux and going to change again.  Just leave it thought